test/spec/service.test.ts

```diff
@@ -146,6 +147,43 @@ describe("Angular service", () => {
       return service;
     }
 
+    describe('$authenticationState', () => {
```

Comment:
Can you add another test case here where the authState is updated before subscribe is called. The test should verify that the correct updated state (not false) is passed.